### ISSUE TITLE
Add others init commands

### ISSUE THREAD

**Issue by horaciod** [mevdschee/php-crud-api#879]
is there a chance to add more initcommands via config ? 
I made this little hack 
```php
case 'pgsql':
                   return [
                        "SET NAMES 'UTF8';",
//hack for set search_path for postgresql 
                        'set search_path to biblio,sigsid,public; '
                    ];
```
Is working very well for me but maybe it can be set with config 

something like 
```php 

    $config = new Config([
        'driver' => 'pgsql', .. 
      'initcommand'=> 'set search_path to biblio,sigsid,public;  ' 
... 

```
I can provide a PR is you wanted.

**Comment by mevdschee** [comment]
Hi Horacio, Thank you for creating this issue. Your request makes a lot of sense to me. I think it can be implemented quite easily. If you do a PR I'll review and merge it. Kind regards, Maurits

**Comment by mevdschee** [comment]
Released in v2.14.3
